src/LongRange/KContainer.h

```diff
@@ -29,15 +29,16 @@ class KContainer : public QMCTraits
 {
 private:
   //Function to return a unique number for each kVector
-  inline long GetHashOfVec(const TinyVector<int, 3>& inpv, int hashparam)
+/*  inline long long GetHashOfVec(const TinyVector<int, 3>& inpv, int hashparam)
```

Comment:
Do we still need these commented code? Or delete?

Reply:
Oops.  Removed.